docs/roadmap.md

```diff
@@ -1,46 +1,37 @@
-
-### Version 2 release
-
-Over the past year, we have gathered a number of feature requests and improvements that we want to implement but that are not backwards compatible with the current API surface. We are going to release our next major version (v2) by the end of 2023, which will include some of these changes:
-
-- [x] **ES Modules support ([#521](https://github.com/aws-powertools/powertools-lambda-typescript/issues/521))** - Thanks to the work of the community we have been able to validate the feasibility of dual support for CommonJS and ES Modules. We are currently working on a plan to implement this.
-- [x] **TypeScript 5.x support ([#1375](https://github.com/aws-powertools/powertools-lambda-typescript/issues/1375))** - This new version of TypeScript brings breaking changes to the experimental decorators API, which we use in our core utilities. We need to investigate the impact of this change and how to best support it.
-- [x] **Support for 3rd party observability providers ([#1500](https://github.com/aws-powertools/powertools-lambda-typescript/issues/1500))** - Many of our customers would like to use Powertools for AWS to send logs, traces, and metrics to providers other than Amazon CloudWatch. At the moment we are investigating the feasibility of this for the Logger utility, while the Python version of Powertools for AWS Lambda is considering this also for Tracer and Metrics.
-
-You can find the full list of changes in the [v2 RFC](https://github.com/aws-powertools/powertools-lambda-typescript/issues/1714) and follow the progress in the [v2 milestone](https://github.com/aws-powertools/powertools-lambda-typescript/milestone/12).
-
-Just like for the feature parity, if you have any feedback or would like to contribute to the discussion, please leave a comment in the RFC or join the discussion in our [Discord community](https://discord.gg/B8zZKbbyET){target="_blank"}.
+- [ ] [RFC to discuss initial thoughts and feasibility for TS/JS ecosystem](https://github.com/aws-powertools/powertools-lambda-typescript/issues/413)
+- [ ] Explore pros & cons of whether to build atop lean frameworks (e.g., Hono) or from scratch
+- [ ] Support for API Gateway REST API resolver
+- [ ] Support for API Gateway HTTP API resolver
+- [ ] Support for Lambda Function URL resolver
+- [ ] Support for Application Load Balancer resolver
+- [ ] Support for VPC Lattice resolver
+- [ ] Support for Data Validation _(e.g., `Zod`)_
+- [ ] Support for OpenAPI generation
+- [ ] Support for Middlewares
+- [ ] Support for Compression
+- [ ] Support for Binary responses
+- [ ] Support for custom serializer
+- [ ] Support for injecting request details _(consider not doing globals like Python legacy)_
+- [ ] Support for Router _(multi-file routes)_
+- [ ] Docs
```

Comment:
I think this item is redundant and we don't need to mention it explicitly. We can also remove it in other sections.

Reply:
Fair. removed it.